samples/stress-tests.sh

```diff
@@ -152,7 +175,7 @@ function tfsimple_join_pipeline() {
       seldon pipeline infer join${1} '{"inputs":[{"name":"INPUT0","data":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"datatype":"INT32","shape":[1,16]},{"name":"INPUT1","data":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"datatype":"INT32","shape":[1,16]}]}'
       seldon pipeline infer join${1} --inference-mode grpc '{"model_name":"simple","inputs":[{"name":"INPUT0","contents":{"int_contents":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]},"datatype":"INT32","shape":[1,16]},{"name":"INPUT1","contents":{"int_contents":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]},"datatype":"INT32","shape":[1,16]}]}'
       unload pipeline join${1} /tmp/pipelines/tfsimples-join${1}.yaml
-      # we cant unload the models here as they are used by other pipelines
+      # we cant unload the models here as they are used by other pipelines ?
```

Comment:
This was for running multiple smoke tests in parallel, driving load to the same models. (just for context)